yarn.lock

```diff
@@ -4266,12 +4271,7 @@ abab@^2.0.3, abab@^2.0.5, abab@^2.0.6:
   resolved "https://registry.yarnpkg.com/abab/-/abab-2.0.6.tgz#41b80f2c871d19686216b82309231cfd3cb3d291"
   integrity sha512-j2afSsaIENvHZN2B8GOpF566vZ5WVk5opAiMTvWgaQT8DkbOqsTfvNAvHoRGU2zzP8cPoqys+xHTRDWW8L+/BA==
 
-ace-builds@1.4.12:
```

Comment:
!

Reply:
See note in description re: bundle size -- update this to v1.4.14

Reply:
I can pull this into a separate PR since it's not strictly related, but I wanted to offset the size of the node-sql-parser package